src/Products/Signature/Loader/SignatureLoader.cs

```diff
@@ -9,31 +10,35 @@
 namespace GroupDocs.Signature.MVC.Products.Signature.Loader
 {
     /// <summary>
-    /// SignatureLoader
+    /// SignatureLoader.
     /// </summary>
     public class SignatureLoader
     {
-        public string CurrentPath;
-        public Common.Config.GlobalConfiguration globalConfiguration;
+        private string currentPath;
+        private Common.Config.GlobalConfiguration globalConfiguration;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make 'globalConfiguration' 'readonly'.](https://app.codacy.com/manual/GroupDocs/GroupDocs.Signature-for-.NET-MVC/pullRequest?prid=5697000)